src/main/java/org/radarcns/phone/OfflineProcessor.java

```diff
@@ -125,6 +125,7 @@ public final void setInterval(long interval) {
         }
         this.interval = interval;
         if (isStarted.get()) {
+            alarmManager.cancel(pendingIntent);
```

Comment:
It should automatically be cancelled because of the PendingIntent ID

Reply:
true. reverted my changes